### ISSUE TITLE
(FED) Map Module - Keyboard Navigation

### ISSUE THREAD

**Issue by KirstyScott** [visitscotland/dot-com-brxm#83]
**Discussions needed on what functionality we want here** 

Environment: feature
Browser: All

Expected outcome: When a keyboard only user navigates through the map module's side panel, when a section is selected, the sub section of that should be focused on. 

Actual outcome: When navigating with keyboard, the map is focused on after pressing enter on a section. 

Steps to recreate: Navigate to a page that has the new map module and navigate through the side panel with keyboard only - the map will be focused instead of the sub sections.
